plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -90,11 +78,6 @@ public BigQueryClient(BigQuery bigQuery, BigQueryConfig config, ViewMaterializat
             return Optional.of(RemoteDatabaseObject.of(datasetName));
         }
 
-        Optional<RemoteDatabaseObject> remoteDataset = remoteDatasets.getIfPresent(datasetName);
-        if (remoteDataset != null) {
-            return remoteDataset;
-        }
-
         // cache miss, reload the cache
```

Comment:
Can we remove or modify these comment about cache in this class?

Reply:
done